src/OpenRealEstate.Core/Rental/RentalPricing.cs

```diff
@@ -14,6 +14,8 @@ public class RentalPricing
 
         public int? Bond { get; set; }
 
+        public bool DepositTaken { get; set; }

```

Comment:
nitpic: rename to `HasReceivedDeposit`

context: for `bool`'s, I usually prepend it with either `Is` or `Has` etc

Reply:
fixed